src/Tribe/Attendees_Table.php

```diff
@@ -200,6 +200,32 @@ public function column_ticket( $item ) {
 	 * @return string
 	 */
 	public function column_check_in( $item ) {
+		$default_checkin_stati_map = array(
+			'woo' => array( 'completed' ),
```

Comment:
It feels like Woo shouldn't be referenced in this file since it is the generic Attendees Table report - perhaps hook here from event-tickets-plus?